layouts/shortcodes/readfile.md

```diff
@@ -5,7 +5,7 @@
   {{ $.Scratch.Set "filepath" ( .Get "file" ) }}
 {{ end }}
 {{ if eq (.Get "markdown") "true" }}
-  {{ $.Scratch.Get "filepath" | readFile | markdownify }}
+  {{ readFile ($.Scratch.Get "filepath") }}
```

Comment:
Did you mean to lose the `markdownify`?

Reply:
Yes, Im a part of the Docsy open source team now and Hugo recently fixed how it processes shortcodes. For shortcodes called with {{% .... %}} syntax that use %, they all get sent to the markdown processor (blackfriday) so `markdownify` is not needed.

The improvement from the PR is shown here:
- https://deploy-preview-53--knative.netlify.com/docs/

Vs our currently published version that excludes the left-side "in-page" nav tree: https://knative.dev/docs/

In line with our other PR about code, if we call {{< ... >}} the readfile shortcode using <, i think it just copies in the targeted file but I need to test and verify.